docs/guides/insar_product_guide.md

```diff
@@ -191,18 +191,24 @@ All of the phase differences in a wrapped interferograms lie between -π and π.
 Before the interferogram can be unwrapped, it must be filtered to remove noise. This is accomplished using an adaptive spectral filtering algorithm. This adaptive interferogram filtering aims to reduce phase noise, increase the accuracy of the interferometric phase, and reduce the number of interferogram residues as an aid to phase unwrapping. In this case, residues are points in the interferogram where the sum of the phase differences between pixels around a closed path is not 0.0, which indicates a jump in phase.
 
 #### Masking
-Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated from the coherence and amplitude (backscatter intensity) values. 
+Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated by applying thresholds to the coherence and/or amplitude (backscatter intensity) values for an image pair. For On Demand InSAR products, we only use a coherence threshold.
```

Comment:
We use cc threshold =0.1 and intensity threshold =0.1

Reply:
@cirrusasf This is a big surprise to me. I thought that all the products I tested had a power threshold of 0.0.

Reply:
Sorry I mad mistake. You are right. We use cc_0.1 and power_0.0.